package.json

```diff
@@ -59,6 +59,7 @@
     "wait-on": "^5.2.1"
   },
   "scripts": {
+    "postinstall": "sh ./tools/setup/setup_hooks.sh",
```

Comment:
I think this might be helpful for the people of Kiva but could be invasive to the people that just download the package.